src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -488,6 +491,76 @@ void generateOverallSummaryFiles(
     generateSummaryJson(roundTallies, tallyTransfers, null, outputPath);
   }
 
+  // write CastVoteRecords for all contests to the same folder as the csvInputFile
+  // create the folder if needed
```

Comment:
How is it possible that the folder would need to be created if it's the same as the input?